src/scripts/ci/travis/install.sh

```diff
@@ -20,6 +20,9 @@ fi
 if [ "$TRAVIS_OS_NAME" = "linux" ]; then
     sudo apt-get -qq update
 
+    pip install --user pylint
+    pip3 install --user pylint
+
```

Comment:
The pylint run should go into it's own Travis build without a binary, like docs target does.

That way when `pip pylint` install breaks some day, it just fails the pylint sub build, not all Travis builds. And when it fails it is obvious what is breaking.